src/models/elimination_scoreboard.ts

```diff
@@ -0,0 +1,84 @@
+import Scoreboard from "./scoreboard";
+import EliminationPlayer from "./elimination_player";
+import _logger from "../logger";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const logger = _logger("elimination_scoreboard");
+
+export default class EliminationScoreboard extends Scoreboard {
+    protected players: { [userID: number]: EliminationPlayer };
+    private startingLives: number;
+
+    constructor(lives: number) {
+        super();
+        this.startingLives = lives;
+    }
+
+    getScoreboardEmbedFields(): Array<{ name: string, value: string, inline: boolean }> {
+        return Object.values(this.players)
+            .sort((a, b) => b.getLives() - a.getLives())
+            .map((x) => {
+                const heartEmoji = "❤️";
+                const lives = x.getLives() <= 5 ? heartEmoji.repeat(x.getLives()) : `${heartEmoji} x ${x.getLives()}`;
```

Comment:
Showing x number of hearts will add too much clutter with large number of players

Reply:
I'll replace it with the number of lives